terraform/modules/elasticache/redis.tf

```diff
@@ -0,0 +1,18 @@
+#tfsec:ignore:aws-elasticache-enable-backup-retention:TODO CLDC-2679 setup a snapshot retention limit
+resource "aws_elasticache_cluster" "main" {
+  #checkov:skip=CKV_AWS_134:TODO CLDC-2679 setup a snapshot retention limit
+  #TODO CLDC-2679 setup redis replicas with multi-az and automatic failover
+  #TODO CLDC-2682 setup redis logging
+  cluster_id                 = "${var.prefix}-redis"
+  auto_minor_version_upgrade = true
+  apply_immediately          = true

```

Comment:
Do we want apply_immediately to be generally on? (I suspect that at the moment while we're likely to be changing things we do, but later we may not).

Reply:
Indeed for now we do, later probably not (atleast not for prod). I've made a ticket to make sure we consider and change this down the line. See [CLDC-2687](https://dluhcdigital.atlassian.net/browse/CLDC-2687)